### PR TITLE
Add Appveyour for 2.7, 3.3 & 3.4

### PR DESCRIPTION
**Pull request by mindw**
includes 1.5.4 changes from bitbucket

### CODE REVIEW

**Comment by untitaker** [comment]
There are a lot of unnecessary updates here, please minimize those to the actual appveyor file.

**Comment by untitaker** [comment]
Also I don't think you'll ever have to use that Powershell script, see https://github.com/untitaker/python-atomicwrites/blob/master/appveyor.yml

**Comment by mindw** [comment]
@untitaker - That setup is indeed a good reference. Perfectly suitable for a pure python package - the tox.ini is perfectly minimal :). It wont work for binary extensions though 

I'l iterate on this again later today.

**Comment by RonnyPfannschmidt** [comment]
i'll merge this one after i finish syncing the repos

**Comment by untitaker** [comment]
@RonnyPfannschmidt I think @mindw intended to do some changes on it.

**Comment by mindw** [comment]
@untitaker I've removed all binary extension support from the appveyor script.

Please take a look.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  - python -m pip install -U pip
+  - python -m pip install -U wheel
+  - python -m pip install -U tox
+
+  # Check that we have the expected version and architecture for Python
+  - python -c "import sys, os;sys.stdout.write(str(sys.version) + os.linesep)"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - python -m pip list
+
+install: false
```
Comment (untitaker) [inline appveyor.yml]: I think this line breaks the build: It executes the `false` command which will return a bad exit code.

Comment (mindw) [inline appveyor.yml]: Yes, doing stuff at 1 AM tends to do that.


-gabi

**Pull request merged**